docs/fuzzer_configuration.md

```diff
@@ -0,0 +1,109 @@
+# kAFL Fuzzer Configuration
+
+The kAFL fuzzer configuration can be changed via config files, environment variables and command line switches.
+
+The [Dynaconf](https://www.dynaconf.com/) framework is used behind the scenes for configuration management, so
+everything that you learn from Dynaconf's documentation should also be applicable for kAFL fuzzer.
+
+## Configuration files precedence
+
+All configuration files are using the `YAML` format.
+
+1. kAFL fuzzer packaged default configuration file: [`kafl_fuzzer/common/config/default_settings.yaml`](https://github.com/IntelLabs/kafl.fuzzer/blob/master/kafl_fuzzer/common/config/default_settings.yaml)
+2. System-level config file: `/etc/xdg/kAFL/settings.yaml`
+3. User-level config file: `$HOME/.config/kAFL/settings.yaml`
+4. Local config file: `$PWD/kafl.yaml`
+5. Environment variable `KAFL_CONFIG_FILE` if specified
+
+Note: if `KAFL_CONFIG_FILE` is specified but points to a non-existing file, a validation error will be raised (to warn the user).
```

Comment:
The note seems kind of redundant. Folks will see the message when they do the error :-)

Reply:
You would have to run kafl to understand this behavior.
I'd rather document this than let the user to find out.